src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -1253,6 +1261,26 @@ otError RadioSpinel<InterfaceType, ProcessContextType>::Get(spinel_prop_key_t aK
     return error;
 }
 
+template <typename InterfaceType, typename ProcessContextType>
+otError RadioSpinel<InterfaceType, ProcessContextType>::GetWithParam(spinel_prop_key_t aKey,
```

Comment:
is this going to be used by other functions? If not, should it be embedded into CalcRcpTimeOffset() for now?

Reply:
OK.